airflow/providers/apache/hive/plugins/hive.py

```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+from typing import Any
+
+from airflow.providers.apache.hive.macros.hive import closest_ds_partition, max_partition
+
+
+class HivePlugin:
```

Comment:
```suggestion
from airflow.plugins_manager import AirflowPlugin

class HivePlugin(AirflowPlugin):
```

If you do this, you can get rid of all the empty defaults and just set `name` and `macros`. Less fragile?

Reply:
Yeah. I just followed Examples :) .... Maybe worth fixing examples too :)

Reply:
Done.